tests/ClientTest.php

```diff
@@ -68,7 +66,10 @@ public function setupOnce(): void
                 'integrations' => [$integration],
             ]),
             $this->createMock(TransportInterface::class),
-            $eventFactory,
+            null,
+            null,
+            null,
+            null,
```

Comment:
Having put the `null`s default up, there would be no longer necessary

Reply:
Can't because we test something in the logger here 😅

Reply:
I don't understand... I'm just talking about omitting the `null`s here.

Reply:
You missed that there is one last argument after the line you commented on, which is `$logger` and whose value comes from the data provider of the test